src/ipa/ipu3/algorithms/grid.cpp

```diff
@@ -78,6 +78,12 @@ int Grid::configure(IPAContext &context, const IPAConfigInfo &configInfo)
 	return 0;
 }
 
+void Grid::prepare(IPAContext &context, ipu3_uapi_params &params)
+{
+	/* Update the IPU3 parameters with new calculated grid */
```

Comment:
I wouldn't call it the new calculated grid, as ... it's not new on .. the second call to ::prepare().

Perhaps just 
 /* Update the IPU3 parameters with the configured grid */